code/modules/spells/aoe_turf/fall.dm

```diff
@@ -94,7 +94,32 @@ var/global/list/falltempoverlays = list()
 		after_cast(targets) //generates the sparks, smoke, target messages etc.
 		invocation = initial(invocation)
 
-/spell/aoe_turf/fall/cast(list/targets, mob/user, var/ignore_timeless = FALSE)
+/spell/aoe_turf/fall/proc/performignore(mob/user = usr, skipcharge = 1, ignore_path) //G U N K
```

Comment:
Did you copypaste this entire proc twice instead of add an argument?

Reply:
I will fix this. I believe I made a mental note to do that with //GUNK but forgot about it.